gradle.properties

```
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.lib
 version=1.0.3-SNAPSHOT
```

```suggestion
version=1.1.0-SNAPSHOT
```
Don't we need to update this into the next minor version(1.1.0)?